docs/topics/api/promoted.rst

```diff
@@ -0,0 +1,21 @@
+================
+Promoted add-ons
+================
+
+.. note::
+    These APIs are subject to change at any time and are for internal use only.
+
+
+--------------
+Stripe Webhook
+--------------
+
+.. _stripe-webhook:
+
+This endpoint receives `event notifications
+<https://stripe.com/docs/webhooks>`_ from Stripe.
```

Comment:
random comment that doesn't need action: this could all be on one line (the docs don't have line limits strictly enforced)